overwatch/processing/alarms/example.py

```diff
@@ -20,21 +23,34 @@ def __str__(self):
         return self.__class__.__name__
 
 
-def alarmConfig():
-    boarderWarning = BetweenValuesAlarm(minVal=0, maxVal=50, alarmText="WARNING")
-    boarderWarning.addReceiver(printCollector)
+def alarmConfig(recipients):
```

Comment:
probably it will be better create in new file, and keep previous example in this file